Document/0x05g-Testing-Network-Communication.md

```diff
@@ -439,12 +439,12 @@ The default configuration for apps targeting Android 6.0 (API level 23) and lowe
 
 #### Dynamic Analysis
 
-For dynamic analysis by using an interception proxy as Burp you can patch the Network Security Configuration file, as described in the "Setting up a Testing Environment for Android Apps" chapter, section "Bypassing the Network Security Configuration".
+When performing dynamic analysis using an interception proxy such as Burp, you may want to patch the Network Security Configuration file, as described in the "Setting up a Testing Environment for Android Apps" chapter, section "Bypassing the Network Security Configuration".
```

Comment:
this is still very passive , can you voice it more active?

Reply:
I also see now that the chapter _doesn't exist_. I'll correct that as well.